girder/api/v1/notification.py

```diff
@@ -70,7 +70,7 @@ def stream(self, timeout, params):
         setResponseHeader('Cache-Control', 'no-cache')
         since = params.get('since')
         if since is not None:
-            since = datetime.utcfromtimestamp(since)
+            since = datetime.fromtimestamp(since, timezone.utc)
```

Comment:
This changes `since` from being a naive to an aware timestamp. Is that a behavioral change when we pass it into mongodb queries below?

Reply:
By my understanding of https://pymongo.readthedocs.io/en/stable/examples/datetimes.html the answer would be "no, there shouldn't be any behavioral change". I missed conversion to tz_aware during read from database. I will fix it shortly.